src/wwwroot/partials/header.html

```diff
@@ -30,7 +30,8 @@ <h4 ng-if="isPermanentAuthed()"> {{'downloads_header' | translate}} </h4>
       </a>
     </nav>
     <nav ng-if="isPermanentAuthed()">
-      <div click-outside="hideConfigDropDown()">
+      <div click-outside="hideConfigDropDown()" class="right-container">
+        <a href="#/settings/my-plan" class="upgrade--link uppercase"><strong>Upgrade</strong></a>
```

Comment:
It should be hidden when the user is not free, also the URL should throw an error if the user is not free.